unilog/generate_input.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+set -e
+
+
+function dumpText(){
+  echo "this is a sheddable clevel=sheddable"
+  echo "this is a sheddableplus clevel=sheddableplus"
+  echo "this is a critical clevel=critical"
+  echo "this is a criticalplus clevel=criticalplus"
+  echo "this is an ERROR clevel=sleddable"
+  echo "this is a default (sheddableplus)"
+}
+
+
+
+while true; do
+  dumpText
+  sleep 2
+done
+
```

Comment:
Yes! This is intended to generate input logged at different levels, to make it easy to test behavior locally.